src/labelle/lib/constants.py

```diff
@@ -34,20 +34,56 @@
     "WARNING: This device is not confirmed to work with this software. Please "
     "report your experiences in https://github.com/labelle-org/labelle/issues/4"
 )
+
+
+# Supported USB device ID enumeration
+class SUPPORTED_DEVICE_ID(Enum):
+    LABELMANAGER_PC = 0x0011
+    LABELPOINT_350 = 0x0015
+    LABELMANAGER_PC_II = 0x001C
+    LABELMANAGER_PNP_NO_MODE_SWITCH = 0x1001
+    LABELMANAGER_PNP_MODE_SWITCH = 0x1002
+    LABELMANAGER_420P_NO_MODE_SWITCH = 0x1003
+    LABELMANAGER_420P_MODE_SWITCH = 0x1004
+    LABELMANAGER_280P_NO_MODE_SWITCH = 0x1005
+    LABELMANAGER_280P_MODE_SWITCH = 0x1006
+    LABELMANAGER_WIRELESS_PNP_NO_MODE_SWITCH = 0x1007
+    LABELMANAGER_WIRELESS_PNP_MODE_SWITCH = 0x1008
+    MOBILE_LABELER = 0x1009
+
+
+# fmt: off
+# Very bad I know, but it keeps a loop of removing the line breaks and errroring
```

Comment:
Try to use slash at the end of the line that you break, .e.g.:
```
    SUPPORTED_DEVICE_ID.LABELMANAGER_PC: \
        "DYMO LabelMANAGER PC",
```

Reply:
I though I removed this? I can´t find it at my current ¨state"

Reply:
I did a search for backslashes, and all the files you changed are clean. Feel free to resolve this.